search/processor.cpp

```diff
@@ -164,7 +125,8 @@ Processor::Processor(Index & index, CategoriesHolder const & categories,
       {StringUtf8Multilang::kInternationalCode, StringUtf8Multilang::kEnglishCode},
       {StringUtf8Multilang::kDefaultCode}};
 
-  m_keywordsScorer.SetLanguages(langPriorities);
+  m_ranker.m_keywordsScorer.SetLanguages(langPriorities);
+  m_preRanker.m_ranker = &m_ranker;
```

Comment:
This reference is removed for now. The dependency is to be resolved in another PR.